core/src/main/java/org/apache/iceberg/util/ParallelIterable.java

```diff
@@ -61,7 +61,7 @@ private ParallelIterator(Iterable<? extends Iterable<T>> iterables,
       this.tasks = Iterables.transform(iterables, iterable ->
           (Runnable) () -> {
             try (Closeable ignored = (iterable instanceof Closeable) ?
-                (Closeable) iterable : () -> {}) {
+                (Closeable) iterable : () -> { }) {
```

Comment:
Strange. Lots of private constructors use `{}` without a space.

Reply:
I think the rules are different if it's a lambda method body. Not sure though - I filed https://github.com/palantir/gradle-baseline/issues/615 asking about it.